monitoring/notification/dashboard.py

```diff
@@ -0,0 +1,567 @@
+from grafanalib.core import (
+    ConstantInput,
+    DataSourceInput,
+    RowPanel,
+    Threshold,
+    StateTimeline,
+    Templating,
+    Template,
+    Heatmap,
+    HeatmapColor,
+    YAxis,
+)
+from grafanalib import formatunits as UNITS
+from scalgrafanalib import(
+    layout,Target,
+    TimeSeries,
+    Dashboard,
+    Stat,
+    GaugePanel,
+    metrics,
+    Tooltip,
+)
+
+LABEL_EXPR = {
+    'notification-processors': '.*-(.*)-headless',
+    'topics': '.*backbeat-notification-(.*).*'
+}
+
+DEFAULT_JOB_PREFIX = 'artesca-data-'
+
+class Metrics:
+    DELIVERY_LAG = metrics.BucketMetric(
+        'notification_queue_processor_notification_delivery_delay_sec',
+        'target', 'job', 'status', namespace='${namespace}',
+    ).with_defaults(
+        'status="success"',
+        'job=~"${notification_processor}"',
+    )
+
+    CONSUMED_EVENTS = metrics.CounterMetric(
+        'notification_queue_populator_event',
+        'job', namespace='${namespace}'
+    ).with_defaults(
+        'job="${job_notification_producer}"'
+    )
+
+    NOTIF_SIZE, PROCESSOR_EVENTS = [
+        metrics.CounterMetric(
+            name, 'job', *extraLabels, namespace='${namespace}'
+        ).with_defaults(
+            'job=~"${notification_processor}"'
+        ) for name, extraLabels in {
+            'notification_queue_processor_notification_size': ['target'],
```

Comment:
'target' is in each metrics' `extraLabels` : maybe should be moved out of this list